config/deploy.rb

```diff
@@ -37,6 +43,14 @@
 # set :keep_releases, 5
 
 namespace :deploy do
+  desc 'Restart application'
+  task :restart do
+    on roles(:app), in: :sequence, wait: 5 do
+      execute :touch, release_path.join('tmp/restart.txt')
```

Comment:
Yeah, I did see that.  The Passenger-5 style restarts was what kept me away from it (although that was also before we had passenger 5 running on the server).